File: src/main/java/com/hank/Hello.java
package com.hank;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello world");
        String s = "abcde";
        System.out.println(s.charAt(3));
        System.out.println(s.substring(1,4));

        Person p =new Person("Tom",66.5f,1.7f);
        p.hello();
        p.bmi();
        System.out.println(p.bmi());
        int score = 88;
        System.out.println(score< 80 || score > 90);
        char c = 'A';
        System.out.println(c>'a');

        /*int age = 19;
        Integer age2 = 19;
        char c = 'A';
        Character c2 ='A';
        byte b = 120;
        float weight =66.5f;
        double height = 1.7;
        boolean adult = true;
        String name = "Tom";*/
    }
}

File: src/main/kotlin/com/hank/Hello.kt
package com.hank

fun main() {
//    println("Hello Kontlin")
//    Human().hello()
    var s:String? = "abcde"
//    s = null
    println(s?.length)
    println(s?.get(3))
    val h=Human(weight = 66.5f,height = 1.7f)
    println(h.bmi())
    val score = 88
    println(score>60)
    val c : Char = 'A'
    println(c.toInt() > 60)
    /*val age = 19
    var age2 = 18
    age2 = 20
    var weight = 66.5f
    val name = "Hank"
    val name: String name="Hank"*/
}
class Human(var name:String = "",var weight:Float,var height:Float){
    init {
        println("test $weight")
    }
//    constructor(name:String,weight: Float,height: Float):this(weight,height)
    fun bmi():Float{val bmi = weight/(height*height)
    return bmi}
    fun hello(){
        println("Hello kotlin")
    }
}